tardis/widgets/sdec_plot.py

```diff
@@ -876,10 +876,14 @@ def generate_plot_mpl(
         self.ax.set_xlabel(r"Wavelength $[\AA]$", fontsize=12)
```

Comment:
this x-axis unit is still italicised

Reply:
Yes, I will do that too but the problem didn't seem to occur in the X-axis though I will hard code it that way :)